src/__tests__/unit/controllers/redirect/test.tsx

```diff
@@ -20,7 +20,7 @@ const MockHistory = {
   goForward: jest.fn(),
   block: jest.fn(),
   registerBlock: jest.fn(),
-  listen: jest.fn(),
```

Comment:
The listen mock was wrong as it has to return an unsubscribe function otherwise `RouterSubscriber` will call `listen()` action at every render and cause a render loop. Discovered that now as prev sweet-state version was over-optimising subscribers, memoizing even when not using selectors (but the hook version didn't) but it was only by luck.